component/src/test/java/com/celements/blog/article/ArticleParentTest.java

```diff
@@ -0,0 +1,74 @@
+package com.celements.blog.article;
+
+import static org.easymock.EasyMock.*;
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.xwiki.model.reference.DocumentReference;
+
+import com.celements.blog.service.IBlogServiceRole;
+import com.celements.common.test.AbstractBridgedComponentTestCase;
+import com.celements.parents.IDocParentProviderRole;
+import com.xpn.xwiki.XWiki;
+import com.xpn.xwiki.XWikiContext;
+import com.xpn.xwiki.web.Utils;
+
+public class ArticleParentTest extends AbstractBridgedComponentTestCase {
+  private XWikiContext context;
+  private XWiki xwiki;
+  private IBlogServiceRole blogServiceMock;
+  private ArticleParent articleParentProvider;
+
+  @Before
+  public void setUp_ArticleParentTest() throws Exception {
+    context = getContext();
+    xwiki = createMock(XWiki.class);
+    context.setWiki(xwiki);
```

Comment:
Use existing AbstractBridgedComponentTestCase.getWikiMock() instead of creating own mock and setting in context.